dataactvalidator/config/sqlrules/d7_detached_award_financial_assistance.sql

```diff
@@ -0,0 +1,8 @@
+-- URI is a required field for aggregate records (i.e., when RecordType = 1)
+SELECT
+    row_number,
```

Comment:
Ahh, I included it in d1 for the same error for the fain because technically we're checking against record_type as well. It's not an error if record_type is 2 so I thought it would be good to have that there for them to see